Src/Extern/SUNDIALS/AMReX_Sundials.cpp

```diff
@@ -0,0 +1,56 @@
+#include <AMReX_Sundials.H>
+
+namespace amrex {
+namespace sundials {
+
+namespace {
+  std::vector<bool> initialized;
+  std::vector<::sundials::Context*> the_sundials_context;
```

Comment:
Is there a strong reason not to use `amrex::Vector` here? 
https://github.com/AMReX-Codes/amrex/blob/135010c8baab688b9a190f008e5098a50a91edb2/Src/Base/AMReX_Vector.H#L18-L22

Reply:
Note that `Vector<bool>` does not work.  If you choose to use `Vector`, you can use `Vector<int>` instead.

Reply:
I switched this over to `Vector<int>`